Makefile

```diff
@@ -25,6 +27,14 @@ clean:
 	helm delete orkestra -n orkestra 2>&1
 	kind delete cluster --name orkestra 2>&1
 
+# Build the docker image and push it to local docker registry
+local-build:
+	docker build . -t localhost:5000/orkestra:dev
```

Comment:
We would need to rely on the KinD cluster's local registry
The instructions are here to setup KinD with a local-registry and then that can be used for storing and pulling images : https://kind.sigs.k8s.io/docs/user/local-registry/